sqlglot/dialects/oracle.py

```diff
@@ -142,6 +142,9 @@ def offset_sql(self, expression: exp.Offset) -> str:
         def table_sql(self, expression: exp.Table, sep: str = " ") -> str:
             return super().table_sql(expression, sep=sep)
 
+        def subquery_sql(self, expression: exp.Subquery, sep: str = " ") -> str:
```

Comment:
```suggestion
        def subquery_sql(self, expression: exp.Subquery, sep: str = " AS ") -> str:
```

Reply:
I'd suggest the following instead, if we don't want to override the default value:

```python
TRANSFORMS = {
    ...
    exp.Subquery: lambda self, e: self.subquery_sql(e, sep=" ")
    ...
}
```

Same for `exp.Table` (so, we'd get rid of both `subquery_sql` and `table_sql` overrides in Oracle).

Reply:
i'll do it